### ISSUE TITLE
Low contrast gray text used for task editor's contents

### ISSUE THREAD

**Issue by nekohayo** [getting-things-gnome/gtg#1375]
In the git version (that would become 0.7), in the task editor, the title and body text's foreground color is `#504e55` gray instead of pure black or pure white (depending on light/dark theme):

<img width="761" height="429" alt="Image" src="https://github.com/user-attachments/assets/95833efe-29c1-4b9f-9623-091003b3e728" />

This lowers contrast and I would consider it an accessibility problem particularly in light mode.  
See also: https://contrastrebellion.com
